docs/index.css

```diff
@@ -46,3 +46,55 @@ code {
 .Demo ul {
   margin: var(--space-large);
 }
+
+.Footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  flex: none;
+  flex-direction: row;
+  box-sizing: border-box;
+  z-index: var(--z-index-sidebar);
+  display: flex;
+  align-items: center;
+  height: 50px;
+  border-width: 1px 0 0 0;
+  border-style: solid;
+  border-color: #b3bfc6;
+  background-color: var(--white);
```

Comment:
I was confused why this was here for a second, and then I realized this was the background color for the light theme.

A cleaner way to do this would be to set vars for each theme and utilize them in the styles:

```css
.cauldron--theme-dark .Footer {
  --footer-background-color: var(--accent-medium);
}

.cauldron--theme-light .Footer {
  --footer-background-color: #fff;
}

.Footer {
  background-color: var(--footer-background-color);
}
```

The var will cascade down appropriately based on the current "theme". This would be recommended for any style that's theme specific.

Reply:
Could this be a different PR? I was trying to avoid editing anything outside the docs, but now that we know there is a UXPin that includes a footer, we could make an issue to make it one of the Cauldron components.

Reply:
I like @scurker's suggestion. I guess it's up to you if you wanted to do it here or put it in another PR.

Reply:
🤣 I misunderstood. I thought this meant add variables to the Cauldron package, not in the same file.